improve clarity and structure of org mgmt docs (#850)
DevCycleHQ/devcycle-docs: docs/platform/account-management/organizations.md

sidebar_position: 1
---

Organizations are the top level of the Account hierarchy within DevCycle, and are where all Users, Projects, and settings are managed. Users within an organization have the same [permissions](/platform/security-and-guardrails/permissions) across all projects, and a user may be part of multiple Organizations.

:::tip

[Granular permissions](/platform/security-and-guardrails/permissions) are only available on Business or Enterprise plans. If you'd like to turn on permissions, contact [support](mailto:support@devcycle.com).

:::

To view or switch Organizations, click your user avatar in the top-right corner of the DevCycle Dashboard. The active Organization will be marked with a green badge.

:::caution
Please note that organizational settings cannot currently be viewed or changed through the CLI and require use of the dashboard.
:::
 
## Discovering Other Organizations

> If **Organization Discoverability** is enabled (it is by default), DevCycle will automatically check if any existing Organizations were created by others within your company or domain when a new user signs up. If a match is found, the user will be presented with the option to request access to an existing Organization or create a new one.

Reviewer: ```suggestion
If **Organization Discoverability** is enabled (it is by default), DevCycle will automatically check if any existing Organizations were created by other users with the same email domain when a new user signs up. If a match is found, the user will be presented with the option to request access to an existing Organization or create a new one.
```
